Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM node:6
+FROM node:8-stretch
 MAINTAINER Kitware, Inc. <kitware@kitware.com>
 
 EXPOSE 8080
 
 RUN mkdir /girder
 RUN mkdir /girder/logs
 
-RUN apt-get update && apt-get install -qy software-properties-common python-software-properties && \
```

Comment:
It doesn't seem that these packages are needed now (they go all the way back to the first Dockerfile), but let me know if there's some crucial reason that they are required.

Reply:
I think these are an artifact of originally using a PPA to install node.